### PR TITLE
Add tutorials

### PR DESCRIPTION
**Pull request by arturo-seijas**

### CODE REVIEW

File: docs/tutorials/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+In this tutorial, we'll walk you through the process of deploying the indico charm, relating it to the nginx-ingress-integrator charm, the postgresql-k8s charm and the redis-k8s charm, and inspecting kubernetes resources it creates. We'll then also look at changing the charm configuration via a juju configuration update.
```
Comment (mthaddon) [inline docs/tutorials/getting-started.md:1]: s/indico/discourse/

File: docs/tutorials/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+You will need:
+
+* A laptop or desktop running Ubuntu (or you can use a VM).
+* [Juju and Microk8s](https://juju.is/docs/olm/microk8s) installed. We’ll also want to make sure the ingress add-on is enabled, which we can do by running `microk8s enable ingress`.
+
+## Deploy this charm
+
+Discourse requires connections to PostgreSQL and Redis, so those will be deployed too and related to the Discourse charm. For more information, see the [Charm Architecture](https://charmhub.io/discourse-k8s/docs/charm-architecture).
+
+All the above charm will the deployed in a new model names `discourse`:
```
Comment (mthaddon) [inline docs/tutorials/getting-started.md:14]: s/charm/charms/

File: docs/tutorials/getting-started.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+juju add-model discourse
+
+# Deploy the charms
+juju deploy redis-k8s
+juju deploy postgresql-k8s
+juju deploy discourse-k8s
+
+# Relate redis-k8s and postgresql-k8s to discourse-k8s
+juju relate redis-k8s discourse-k8s
+# For postgresql-k8s the interface db needs to be specified as the charm provides more than one
```
Comment (mthaddon) [inline docs/tutorials/getting-started.md:27]: I'd put quotes around "db" and put it before "interface" as it'll read a little better that way:
```
For postgresql-k8s the "db" interface needs to be specified as the charm provides more than one
```

File: docs/tutorials/getting-started.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+redis-k8s/0*       active    idle   10.1.180.78  6379/TCP  
+
+Relation provider    Requirer             Interface  Type     Message
+postgresql-k8s:db    discourse-k8s:db     pgsql      regular  
+postgresql-k8s:peer  postgresql-k8s:peer  peer       peer     
+redis-k8s:redis      discourse-k8s:redis  redis      regular  
+
+```
+
+Run `kubectl get pods -n discourse` to see the pods that are being created by the charms
```
Comment (mthaddon) [inline docs/tutorials/getting-started.md:54]: I think we should have either a trailing ":" or "." here.

File: docs/tutorials/getting-started.md [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+modeloperator-7879f68947-s4q59   1/1     Running   0          10m
+postgresql-k8s-operator-0        1/1     Running   0          10m
+redis-k8s-operator-0             1/1     Running   0          10m
+redis-k8s-0                      1/1     Running   0          10m
+postgresql-k8s-0                 1/1     Running   0          10m
+discourse-k8s-0                  2/2     Running   0          9m
+
+```
+
+In order to expose the charm, the Nginx Ingress Integrator is to be deployed alongside Discourse to provide ingress capabilities
```
Comment (mthaddon) [inline docs/tutorials/getting-started.md:66]: Same as above - let's have a trailing ":" or ".".

File: docs/tutorials/getting-started.md [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+
+```
+juju deploy nginx-ingress-integrator
+# If your cluster has RBAC enabled you'll be prompted to run the following:
+juju trust nginx-ingress-integrator --scope=cluster
+
+juju relate discourse-k8s:ingress nginx-ingress-integrator:ingress
```
Comment (mthaddon) [inline docs/tutorials/getting-started.md:73]: You don't need to specify the ingress relation here, can be simplified to:
```
juju relate discourse-k8s nginx-ingress-integrator
```

File: docs/tutorials/getting-started.md (hunk 1 above)
Comment (arturo-seijas) [reply docs/tutorials/getting-started.md]: Fixed. Thx

File: docs/tutorials/getting-started.md (hunk 6 above)
Comment (arturo-seijas) [reply docs/tutorials/getting-started.md]: Fixed

File: docs/tutorials/getting-started.md (hunk 5 above)
Comment (arturo-seijas) [reply docs/tutorials/getting-started.md:66]: Fixed

File: docs/tutorials/getting-started.md (hunk 4 above)
Comment (arturo-seijas) [reply docs/tutorials/getting-started.md]: Fixed

File: docs/tutorials/getting-started.md (hunk 3 above)
Comment (arturo-seijas) [reply docs/tutorials/getting-started.md]: Fixed

File: docs/tutorials/getting-started.md (hunk 2 above)
Comment (arturo-seijas) [reply docs/tutorials/getting-started.md]: Fixed

**Review by merkata — DISMISSED** [review]
Looks good!

File: docs/tutorials/getting-started.md [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+You will need:
+
+* A laptop or desktop running Ubuntu (or you can use a VM).
+* [Juju and Microk8s](https://juju.is/docs/olm/microk8s) installed. We’ll also want to make sure the ingress add-on is enabled, which we can do by running `microk8s enable ingress`.
+
+## Deploy this charm
+
+Discourse requires connections to PostgreSQL and Redis, so those will be deployed too and related to the Discourse charm. For more information, see the [Charm Architecture](https://charmhub.io/discourse-k8s/docs/charm-architecture).
+
+All the above charms will the deployed in a new model names `discourse`:
```
Comment (mthaddon) [inline docs/tutorials/getting-started.md:14]: s/names/named/

File: docs/index.md [hunk 8]
```diff
@@ -2,6 +2,6 @@ Discourse is a from-scratch reboot, an attempt to reimagine what a modern Intern
 
 Discourse is a JavaScript application that runs in your web browser, using the [Ember.js](https://emberjs.com/) framework. The server side of Discourse is [Ruby on Rails](https://rubyonrails.org/) backed by a [Postgres](https://www.postgresql.org/) database and [Redis](https://redis.io/) cache. The code is licensed under the [Apache License, version 2](https://www.apache.org/licenses/LICENSE-2.0), and pull requests are accepted once you've signed a [Contributor License Agreement](https://en.wikipedia.org/wiki/Contributor_License_Agreement).
 
-For more information on Redis and PostgreSQL relations see [this section](/t/discourse-documentation-database-relations/3776).
+For more information on Redis and PostgreSQL relations see [this section](https://discourse.charmhub.io/t/discourse-documentation-database-relations/3776).
```
[the remark this answers is not in the document]
Comment (arturo-seijas) [reply docs/index.md]: Indeed. Fixed

File: docs/tutorials/getting-started.md (hunk 7 above)
Comment (arturo-seijas) [reply docs/tutorials/getting-started.md]: Fixed

**Review by mthaddon — APPROVED** [review]
LGTM, thx

**Review by amandahla — APPROVED** [review]
LGTM

**Pull request merged**